### ISSUE TITLE
AttributeError: 'Revision' object has no attribute 'parents'

### ISSUE THREAD

**Issue by glandium** [mystor/phlay#50]
When uploading https://phabricator.services.mozilla.com/D24830?id=81713 with current master, I got the following error:
```
Traceback (most recent call last):
  File "/home/glandium/phlay/phlay", line 931, in <module>
    main()
  File "/home/glandium/phlay/phlay", line 917, in main
    process_args(args)
  File "/home/glandium/phlay/phlay", line 846, in process_args
    if commit.revision is None or pphids != commit.revision.parents(conduit):
AttributeError: 'Revision' object has no attribute 'parents'
```

I was able to do the upload by reverting to f50196fe6221278722b6deaebe225cdc5134f424.

**Comment by glandium** [comment]
Should it be `edges` instead of `parents`?

**Comment by amccreight** [comment]
I experienced this same issue. I was pushing to update a patch that had been landed and reverted, if that matters.

**Comment by mystor** [comment]
I've pushed an update - sorry for missing this update until now. Can you try again with the latest version?

**Comment by mystor** [comment]
(should be fixed in 595288f00c32443ca66d044cbf9fd56bc9748bf6)

**Comment by ehsan** [comment]
Yes, it fixed the issue for me.  Thanks!
